radix-engine-interface/src/model/package/invocations.rs

```diff
@@ -12,6 +12,8 @@ use sbor::rust::string::String;
 pub struct PackagePublishNoOwnerInvocation {
     pub code: Blob,
     pub abi: Blob,
+    pub royalty_config: HashMap<String, RoyaltyConfig>,
```

Comment:
Can we add a TODO: comment that we need to verify with product to what extent do we want royalty configuration to be customizable?

Reply:
Just to clarify: do you mean the methods available for royalty customisation? or the mutability of royalty configuration, like fully mutable/partially mutable/immutable?

Reply:
I mean configurability of auth to claim/update royalties.

Reply:
Decided to remove `royalty_config` and `access_rules_chain` for package no owner route; and remove `access_rules_chain` for package with owner route.

Reply:
As we're removing owner badge as a whole, we need to change this parameter to a `NonFungibleAddress`. I will do it after the removal if you don't mind.